src/modules/metrics/metrics.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { Counter, Gauge, Registry } from 'prom-client';
+
+@Injectable()
+export class MetricsService {
```

Comment:
Der Service für Metrics + Reporter vielleicht in einer eigenen Datei und nicht alles unter metrics.